public/maps/tamil-nadu.json

```diff
@@ -1556,7 +1556,7 @@
     "translate": [76.233101, 8.077847]
   },
   "objects": {
-    "tamilnadu_district": {
+    "tamil-nadu_district": {
```

Comment:
All the files except `tamil-nadu.json` the districts' data under `<file_name>_district` property. This change fixes that.

Reply:
Hey, you'll have to make this change in mapexplorer.js as well. Also, map geojsons are taken from a different repository, so we would have to rename them every time they're updated. So please don't make this change unless it's somehow necessary for the tests.

Reply:
@shuklaayush I thought about it and then forgot.
Fixed it with https://github.com/covid19india/covid19india-react/pull/986/commits/18978a5c0f90bc3a762aa0e1d56439e3ce2846c3. Thanks

I will submit a PR in @guneetnarula repo as well. Thanks for bringing my attention to that too.

Good catches, man! :+1:

Reply:
@shuklaayush over there the name of the file is `tamilnadu.json` and we have `tamil-nadu.json` so do you think we need to rename the file to `tamilnadu.json`?